bundle-workflow/Jenkinsfile

```diff
@@ -75,14 +68,15 @@ pipeline {
                     post() {
                         always {
                             cleanWs disableDeferredWipeout: true, deleteDirs: true
-                            sh "rm -rf ~/.m2/"
                         }
                     }
                 }
                 stage('build-arm64') {
                     agent {
-                        node {
-                            label 'Jenkins-Agent-al2-arm64-m6gxlarge'
+                        docker {
+                            label 'Jenkins-Agent-al2-arm64-c6g4xlarge-Docker-Host'
+                            image 'opensearchstaging/ci-runner:al2-x64-arm64-jdk14-node10.24.1-20210924'
```

Comment:
How do we version the docker image?  Do we need to keep updating the Jenkinsfile?

Reply:
or add the latest tag to remain updated?

Reply:
I dont see that as an issue, just like we keep updating manifest with different branches.